### DISCUSSION TITLE
DNS_PROBE_FINISHED_NO INTERNET and DNS_PROBE_STARTED

### DISCUSSION THREAD

**Discussion by DavidOvMu23** [pivpn/pivpn#1836] [Support]
I have installed pivpn with the most basic configuration possible and it does not work, I connect from wireguard from the mobile and when I go to do an internet search I get this error: DNS_PROBE_FINISHED_NO INTERNET and DNS_PROBE_STARTED

These are the settings I've put in the installation:
![image](https://github.com/pivpn/pivpn/assets/76009842/36901ea4-25a6-4029-8e11-9e49fb67df42)

![image](https://github.com/pivpn/pivpn/assets/76009842/5354db4b-90d4-46ad-8777-a20627bb379a)

![image](https://github.com/pivpn/pivpn/assets/76009842/c054ae7a-fe27-464d-8faf-9bf4fcf535a3)

![image](https://github.com/pivpn/pivpn/assets/76009842/c12127f7-e3b5-4588-bbdb-0ef8b05df4e4)

![Sin título](https://github.com/pivpn/pivpn/assets/76009842/fc78b763-4798-4ea0-8cb0-5462e8699a7a)

And this is the router configuration:
![IMG_20240422_125407](https://github.com/pivpn/pivpn/assets/76009842/3590c6d9-7b75-4a22-be73-b9cd79a356d0)

I've been trying to fix this for literally a week and I haven't gotten anywhere. I don't know what I'm leaving 

Mainly I wanted to make it look like pi-hole and put a ddns of my own, but it doesn't go with anything, I've tried multiple things and I haven't gotten anything, I don't know if it's some error from the last piv updateMainly I wanted to make it look like pi-hole and put a ddns of my own, but it doesn't go with anything, I've tried multiple things and I haven't gotten anything, I don't know if it's some error from the last pivpn update

**Comment by orazioedoardo** [comment]
External port is incorrect, it's 51826 instead of 51820

**Comment by DavidOvMu23** [comment]
the server no works because i'm in a cg-nat
